modules/Bio/EnsEMBL/Analysis/Hive/Config/RepeatMasking.pm

```diff
@@ -0,0 +1,549 @@
+    'databases_server'          => '', # host for general output dbs
+    'dna_db_server'             => '', # host for dna db
+    'pipe_db_port'              => '', # port for pipeline host
+    'databases_port'            => '', # port for general output db host
+    'dna_db_port'               => '', # port for dna db host
+    'repbase_logic_name'        => '', # repbase logic name i.e. repeatmask_repbase_XXXX, ONLY FILL THE XXXX BIT HERE!!! e.g primates
+    'repbase_library'           => '', # repbase library name, this is the actual repeat repbase library to use, e.g. "Mus musculus"
+    'release_number'            => '' || $self->o('ensembl_release'),
+    'species_name'              => '', # e.g. mus_musculus
+    'output_path'               => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'use_genome_flatfile'       => '1',# This will read sequence where possible from a dumped flatfile instead of the core db
+    'skip_repeatmodeler'        => '0', # Skip using our repeatmodeler library for the species with repeatmasker, will still run standard repeatmasker
+    'replace_repbase_with_red_to_mask' => '0', # Setting this will replace 'full_repbase_logic_name' with 'red_logic_name' repeat features in the masking process
+    'red_logic_name'                   => 'repeatdetector', # logic name for the Red repeat finding analysis
+    # Keys for custom loading, only set/modify if that's what you're doing
+    'repeatmodeler_library'        => '', # This should be the path to a custom repeat library, leave blank if none exists
+    'use_repeatmodeler_to_mask'    => '0', # Setting this will include the repeatmodeler library in the masking process
+    
+########################
+# Pipe and ref db info
+########################
+
+    # The following might not be known in advance, since the come from other pipelines
+    # These values can be replaced in the analysis_base table if they're not known yet
+    # If they are not needed (i.e. no projection or rnaseq) then leave them as is
+
+    'pipe_db_name'                  => $self->o('dbowner').'_'.$self->o('production_name').'_pipe_'.$self->o('release_number'),
+    'dna_db_name'                   => $self->o('dbowner').'_'.$self->o('production_name').'_core_'.$self->o('release_number'),
+
+    'reference_db_name'            => $self->o('dna_db_name'),
+    'reference_db_server'          => $self->o('dna_db_server'),
+    'reference_db_port'            => $self->o('dna_db_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'              => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+    'production_db_server'         => 'mysql-ens-meta-prod-1',
+    'production_db_port'           => '4483',
+    
+    databases_to_delete => ['reference_db'],
```

Comment:
I forgot this, you can simply use an empty array because the pipeline is not creating any database. Any database in this array can be deleted if someone uses drop_databases on the command line. But this is not something we want to happeb

Reply:
Requested change effected